crates/subspace-service/Cargo.toml

```diff
@@ -88,6 +88,7 @@ subspace-kzg = { version = "0.1.0", path = "../../shared/subspace-kzg" }
 subspace-networking = { version = "0.1.0", path = "../subspace-networking" }
 subspace-proof-of-space = { version = "0.1.0", path = "../subspace-proof-of-space" }
 subspace-runtime-primitives = { version = "0.1.0", path = "../subspace-runtime-primitives" }
+subspace-sync = { version = "0.1.0", path = "../../shared/subspace-sync" }
```

Comment:
This commit should have been squashed with commit that introduced the crate. Otherwise you have a history disconnect due to file with implementation added there not linked to the file with implementation deleted here, Git will not preserve history of the file because of it, which is annoying.